src/tools/dart-fix.md

```diff
@@ -31,4 +28,62 @@ To apply the proposed changes, use the `--apply` flag:
 $ dart fix --apply
 ```
 
+## Customizing dart fix
+
+The `dart fix` command applies its fixes
+based on the analysis configuration of the code it runs on. 
```

Comment:
While this is accurate, and better than what we had before, it might still be a little vague in that it begs the question of how the "analysis configuration" impacts fixes. (Is "analysis configuration" a concept that we use elsewhere?)

It might be useful to state explicitly that

> The `dart fix` command only applies fixes when there is a problem identified by a diagnostic. If no diagnostic is being reported, then no fix will be applied. Some diagnostics, such as compilation errors, are implicitly enabled, while others, such as lints, must be explicitly enabled in the analysis options file.
>
> You can sometimes increase the number of fixes that can be applied by enabling additional lints. It should be noted, however, that not all diagnostics have associated fixes that can be applied, so enabling new lints might not change the behavior of `dart fix`.

I'd also move lines 35-36 to after the example.

Reply:
Thanks, incorporated!